opel-agent/README.md

```diff
@@ -38,18 +40,40 @@ helm upgrade otel-coralogix-agent coralogix-charts-virtual/opentelemetry-coralog
   -f override.yaml
 ```
 
+## Send your metrics to Coralogix
+By default, the Prometheus receiver is enabled, it collects metrics on the agent itself, and they are sent to Coralogix. 
+In order to disable it, in addition to the `extraEnvs` section, add the following section to your override.yaml:
+*** It will disable the prometheus receiver and the coralogix exporter from the metrics pipeline.
+ 
+```yaml
+---
+#override.yaml:
+opentelemetry-collector:
+  service:
+    pipelines:
+      metrics:
+        exporters:
+        - logging
+        processors:
+        - memory_limiter
+        - batch
+        receivers:
+        - otlp
+```  
+
 ## Monitoring the agent
-If you have Prometheus configured, with the Prometheus operator, it is recommended to enable the servicemonitor and the prometheusrules offered by this chart. By enabling it, a servicemonitor for the agent will be created, and some default Prometheus alerts will be created.    
+If you have Prometheus configured, with the Prometheus operator, it is recommended to enable the podMonitor and the prometheusRules offered by this chart. 
+In order to enable it By enabling it, the metrics port must be enabled, the podmonitor must be enabled, and the prometheusrules must be enabled [if desired].
 
 ## Coralogix Endpoints
 
-| Region  | Traces Endpoint
-|---------|-----------------------------------------------|
-| USA1	  | `tracing-ingress.coralogix.us:9443`           |
-| APAC1   | `tracing-ingress.app.coralogix.in:9443`       |
-| APAC2   | `tracing-ingress.coralogixsg.com:9443`        |
-| EUROPE1 | `tracing-ingress.coralogix.com:9443`          |
-| EUROPE2 | `tracing-ingress.eu2.coralogix.com:9443`      |
+| Region  | Traces Endpoint                               | Metrics Endpoint	
+|---------|-----------------------------------------------|-----------------------------------------------
+| USA1	  | `tracing-ingress.coralogix.us:9443`           | `otel-metrics.coralogix.us:443`	
```

Comment:
Why do we need to mention the port? this is native SSL one

Reply:
its required according to our exporter docs : 
https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter/coralogixexporter

if I delete it is throws `missing port in address`

Reply:
@amit-mazor Thx